### PR TITLE
UHF-4528: Large amount of units listing

### PR DESCRIPTION
**Pull request by teroelonen**
# Add a way to list large amounts of units on a listing [UHF-4528](https://helsinkisolutionoffice.atlassian.net/browse/UHF-4528) & [UHF-4532](https://helsinkisolutionoffice.atlassian.net/browse/UHF-4532) 
Make it possible for the content producers to tag and list large numbers of units at the same time (for example daycare units).

## What was done
* Added views bulk operations and views bulk edit modules.
* Added Unit type vocabulary for the categorisation.
* Added new field Unit type for TPR-units.
* Changed the TPR-unit listing so that you can do views bulk edit on it.
* Added minimal display for TPR units.

## How to install
* Make sure your instance is up and running on latest dev branch.
    * `git pull origin dev`
    * `make fresh`
* Update the Helfi Platform config and hdbt
    * `composer require drupal/helfi_platform_config:dev-UHF-4528_large_amount_of_units_listing -W && composer require drupal/hdbt:dev-UHF-4528_large_amount_of_units_listing -W`
* Run `make drush-updb drush-cr`

## How to test
* Go to `/admin/structure/taxonomy/manage/unit_type/overview` and add one term there for example _Daycare_
* Go to `/admin/content/integrations/tpr-unit` listing and check that there is action called _Modify values_ on the bottom of the listing. Select some units and do the action _Modify values_ for those units. Pick the term you just created for the _Unit type_ field on the selected units and save.
* Go to check any of the units that you edited if there is the term set on the _Unit type_ field.

## Other PRs
* https://github.com/City-of-Helsinki/drupal-hdbt/pull/270
* https://github.com/City-of-Helsinki/drupal-helfi-kasvatus-koulutus/pull/116

### CODE REVIEW

File: helfi_features/helfi_tpr_config/helfi_tpr_config.install [hunk 1]
```diff
@@ -76,6 +77,11 @@ function helfi_tpr_config_update_dependencies() {
     'helfi_content' => 9017,
   ];
 
+  // TODO
```
Comment (tapsal) [inline helfi_features/helfi_tpr_config/helfi_tpr_config.install:80]: Should this TODO be here?

Comment (teroelonen) [reply helfi_features/helfi_tpr_config/helfi_tpr_config.install]: Should not. Fixed to correct comment.

File: helfi_features/helfi_tpr_config/config/install/core.entity_view_display.tpr_unit.tpr_unit.minimal.yml [hunk 2]
```diff
@@ -0,0 +1,70 @@
+langcode: en
+status: true
+dependencies:
+  config:
+    - core.entity_view_mode.tpr_unit.minimal
+    - field.field.tpr_unit.tpr_unit.field_content
+    - field.field.tpr_unit.tpr_unit.field_hs_front_page
```
[the remark this answers is not in the document]
Comment (teroelonen) [reply helfi_features/helfi_tpr_config/config/install/core.entity_view_display.tpr_unit.tpr_unit.minimal.yml]: Removed the dependency - it is only for KASKO so not something we should have here.

**Review by tapsal — APPROVED** [review]
Great! 🚀

**Pull request merged**